### ISSUE TITLE
setting style via variable

### ISSUE THREAD

**Issue by smeevil** [slime-lang/slime#23]
A case i found :

``` elixir
-style = get_flash(@conn, :email_error) && "" || "display: none"
.alert-box.warning.js-news-letter-error.text-center style=style data-alert="data-alert"
  .js-error-message= get_flash(@conn, :email_error)
```

renders to : 

``` html
<div class="alert-box warning js-news-letter-error text-center" data-alert="data-alert" style="display:" none=""><div class="js-error-message"></div></div>
```

where the problems is `style="display:" none=""` which should be `style="display: none"`

**Comment by Rakoth** [comment]
@doomspork Should I fix it now, or wait until #16 is merged?

**Comment by doomspork** [comment]
@Rakoth you can fix.  I'll release 0.5.0 with your fix and other features.  Once #16 is done I'll ship 0.6.0.
